internal/runtime/pod.go

```diff
@@ -100,7 +100,7 @@ func (builder *PodBuilder) AddContainer(name, image string) *ContainerBuilder {
 			Name:            strings.ToLower(name),
 			Image:           image,
 			Env:             []corev1.EnvVar{},
-			ImagePullPolicy: corev1.PullAlways,
+			ImagePullPolicy: corev1.PullIfNotPresent,
```

Comment:
i observed test speedup once i changed this to `IfNotPresent` 
since tests run in parallel, multiple pods start pulling the same image, and it seemed to slow down the tests.

i also used a "warm up" pod which has containers for each image tests needed. 
```
  containers:    
    - name: c0    
      image: elasticsearch:7.10.1    
    - name: c1    
      image: quay.io/openshift/origin-logging-rsyslog:latest    
    - name: c2    
      image: quay.io/openshift-logging/fluentd:1.14.5    
etc
```
then tests didnt have to wait for pods to pull the images.